storage/migrations/10_roothash.up.sql

```diff
@@ -29,4 +29,17 @@ CREATE INDEX ix_events_roothash
         roothash_runtime IS NOT NULL AND
         roothash_runtime_round IS NOT NULL;
 
+CREATE TABLE chain.roothash_messages (
```

Comment:
people are deploying from unreleased main branch. let's move this to a new migration

Reply:
there's a release now